### PR TITLE
Fix pod filter

### PR DESCRIPTION
**Pull request by iblackman**
Pod filter wasn't bringing any results. Replaced the metric used.

### CODE REVIEW

File: pod-resource-usage.json [hunk 1]
```diff
@@ -1802,5 +1820,5 @@
   "timezone": "browser",
   "title": "Pod Resource Usage",
   "uid": "hYTBQhQiz",
-  "version": 4
+  "version": 2
```
Comment (bcdonadio) [inline pod-resource-usage.json]: Please increase this to 5 instead of decreasing.

**Comment by iblackman** [comment]
Example in APP-Dev:
https://grafana-dev.powerapp.cloud/d/hYTBQhQiz2/pod-resource-usage-v2?orgId=1&refresh=10s

File: pod-resource-usage.json (hunk 1 above)
Comment (iblackman) [reply pod-resource-usage.json]: Fix 11d2051.

**Comment by bcdonadio** [comment]
I believe Prometheus is down on APP-Dev, so I can't test it. Could you take a look at it?

**Pull request merged**